src/main/java/com/playtika/homework2/task1/model/Device.java

```diff
@@ -0,0 +1,36 @@
+package com.playtika.homework2.task1.devices;
```

Comment:
move this class and its inheritors to `package com.playtika.homework2.task1.devices`